src/sharedInterfaces/queryResult.ts

```diff
@@ -28,12 +47,35 @@ export interface QueryResultWebviewState {
     value?: string;
     messages: IMessage[];
     tabStates?: QueryResultTabStates;
+    isExecutionPlan?: boolean;
+    sqlPlanContent?: string;
+    executionPlan?: ep.GetExecutionPlanResult;
+    executionPlanGraphs?: ep.ExecutionPlanGraph[];
+    theme?: string;
```

Comment:
iirc the webview state already has theme field

Reply:
this is a theme that specifically says whether it's dark or light, no just the hex code

Reply:
https://github.com/microsoft/vscode-mssql/blob/main/src/reactviews/pages/QueryResult/queryResultUtils.ts
Here's how we use `context.theme` to provide different icons for dark and light themes.